OctoKit/Reactions.swift

```diff
@@ -0,0 +1,23 @@
+/// `Reactions` are available on all Issues and Pull Requests.
+///
+/// `Reactions` are in general available as an object in some API response,
+/// there is no API available directly for `Reactions`.
+public struct Reactions: Codable {
+    public let totalCount: Int?
+    public let plus1: Int?
```

Comment:
I'm on the fence here. This looks super weird but maps the response correctly. Maybe thumbs up/down is more readable?

Reply:
Your suggestion makes sense. I do not think GitHub will change the syntax of `+1` to something different than the thumbs-up.